scripts/pipeline.groovy

```diff
@@ -284,24 +284,12 @@ def uploadBlobs() {
         uploadArtifactsOnlyIfSuccessful: false,
         uploadZips: false,
         virtualPath: "ArtifactsFor-${env.BUILD_NUMBER}/${commitHash}/",
+        storageType: 'blobstorage'
     ])
 }
 
 def downloadBlobs() {
-    step([
-        $class: "AzureStorageBuilder",
-        downloadType: [
-            value: "container",
-            containerName: "skiasharp-public-artifacts",
-        ],
-        includeFilesPattern: "ArtifactsFor-${env.BUILD_NUMBER}/${commitHash}/**/*",
-        excludeFilesPattern: "",
-        downloadDirLoc: "",
-        flattenDirectories: false,
-        includeArchiveZips: false,
-        strAccName: "credential for xamjenkinsartifact",
-        storageCredentialId: "fbd29020e8166fbede5518e038544343",
-    ])
+    azureDownload containerName: 'skiasharp-public-artifacts', downloadType: 'container', fileShare: '', includeFilesPattern: 'ArtifactsFor-${env.BUILD_NUMBER}/${commitHash}/**/*', storageCredentialId: 'fbd29020e8166fbede5518e038544343'
```

Comment:
Is it possible to wrap these lines? Not essential at all, just makes it better for reading.

I was always scared since no semicolons are being used in groovy. where does one line end and another start? This is like python where anything goes! 😜